src/ipc-handlers.ts

```diff
@@ -205,6 +212,13 @@ export async function createSite(
 		}
 	}
 
+	if ( details.phpVersion ) {
+		bumpStat( StatsGroup.STUDIO_SITE_VERSIONS, getPHPVersionMetric( details.phpVersion ) );
+	}
+	if ( wpVersion ) {
+		bumpStat( StatsGroup.STUDIO_SITE_VERSIONS, getWordPressVersionMetric( wpVersion ) );
+	}
```

Comment:
If we are going to track this data, wouldn't it make sense to also track how many sites opt out of selecting a specific WP or PHP version?